src/components/Container.js

```diff
@@ -368,7 +379,19 @@ const mapStateToProps = (state, {contextId, targetVerseText, sourceVerse}) => {
       alignedTokens: getVerseAlignedTargetTokens(state, chapter, verse),
       verseAlignments: getVerseAlignments(state, chapter, verse),
       verseIsValid: getIsVerseValid(state, chapter, verse,
-        normalizedSourceVerseText, normalizedTargetVerseText)
+        normalizedSourceVerseText, normalizedTargetVerseText),
+      groupMenu: {
+        toolsReducer: ownProps.tc.toolsReducer,
+        groupsDataReducer: ownProps.tc.groupsDataReducer,
+        groupsIndexReducer: ownProps.tc.groupsIndexReducer,
+        groupMenuReducer: ownProps.tc.groupMenuReducer,
+        translate: ownProps.translate,
+        actions: ownProps.tc.actions,
+        isVerseFinished: (chapter, verse) => getIsVerseFinished('wordAlignment', ownProps, chapter, verse),
```

Comment:
The tool's API is available under `this.props.toolAPI`. A tool's own API is not wrapped in in an `ApiController` so you can call the methods on it directly. See https://github.com/translationCoreApps/tc-tool/blob/master/TOOL_API.md for details.

Therefore, this line should read as follows:
```js
isVerseFinished: ownProps.toolAPI.getIsVerseFinished,
```

Reply:
Done.